Header header
float64 poi_depth
float64 poi_angle
bool poi_deviation
string poi_label